source/source/nebula/ecs/system/DrawSpriteSystem.hx

```diff
@@ -28,17 +28,13 @@ class DrawSpriteSystem extends AbstractSystem
             {
                 if(sprite.isRepeating)
                 {
-                    var s:FlxBackdrop = new FlxBackdrop(sprite.image);
-                    sprite.sprite =  s;
-                    this.state.add(s);
+                    sprite.sprite = new FlxBackdrop(sprite.image) ;
                 }
                 else
                 {
-                    var s:FlxSprite = new FlxSprite();
-                    s.loadGraphic(sprite.image);
```

Comment:
I think you need to add this line back to load the image if it's not repeating. (Are you testing your changes in the main app via `lime test neko`?)